index.html

```diff
@@ -293,14 +295,15 @@ <h4 class="interest-text">I'm always interested in hearing about new projects, s
 
           <div class="form-fields">
             <label id="mylabel" for="fullname" hidden>Full Name</label>
-            <input type="text" class="full-name" id="fullname" name="name" placeholder="Full name" maxlength="30" required>
+            <input type="text" class="full-name" id="fullname" name="name" placeholder="Full name" maxlength="30">
             <hr class="end-name">
             <label id="mylabel" for="email-address" hidden>Email Address</label>
-            <input type="email" class="email-address" id="email-address" name="email" placeholder="Email address" pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$" required>
+            <input type="email" class="email-address" id="email-address" name="email" placeholder="Email address">
             <hr class="end-address">
             <label id="mylabel" for="write-message" hidden>Write something here</label>
-            <textarea class="text-area" id="write-message" name="message" rows="7" cols="50" maxlength="500" placeholder="Write me something..." required></textarea>
+            <textarea class="text-area" id="write-message" name="message" rows="7" cols="50" maxlength="500" placeholder="Write me something..."></textarea>
             <button class="form-button" type="submit">Get in touch</button>
```

Comment:
[Optional] Please add  `required` for all your input values so as to enable a user fill all input values before submitting a form 😄